lightning/src/ln/data_persister.rs

```diff
@@ -0,0 +1,221 @@
+//! Logic for persisting data from ChannelMonitors on-disk. Per-platform data
+//! persisters are separated into the lightning-persist-data crate. These
```

Comment:
Didn't catch the reason why `LinuxPersister` was moved back into the `lightning` crate. Are there no longer plans for a separate crate?

Reply:
I was going to bring it up in the meeting, but it's hard to do thorough end-to-end tests using all of our nice test utils, without having `LinuxPersister` being in the same crate. I'm not opposed to later dealing with those issues and creating a separate crate, but as it stands I'm not sure if it's justified since it's just one small file (which is another reason why putting off a separate crate might make sense).

Reply:
Update: discussed offline, decided a separate crate is best for reasons explained in the relevant commit message.